content/sensu-go/5.20/reference/assets.md

```diff
@@ -368,6 +371,25 @@ example      | {{< highlight shell >}}
 }
 {{< /highlight >}}
 
+## Asset filters based on entity.system attributes
+
+Use the [entity.system attributes][10] in asset [filters][42] to specify which systems and configurations the asset can be used with.
+
+For example, if your asset is compiled for Linux AMD64 and 386 architectures, you can add filters for `os` and `arch` to your asset:
+
+{{< highlight shell >}}
+"filters": ["entity.system.os=='linux'", "entity.system.arch=='amd64'", "entity.system.arch=='386'"] 
+{{< /highlight >}}
+
+In this example, for Linux systems with either AMD64 or 386 architectures, the `os` filter expression and at least one of the `arch` filter expressions will evaluate as `true`.
```

Comment:
All of these are AND'ed together and all have to be true.  Plus you will only ever have one arch, therefore this example would never work.